packages/validator/src/spectral/index.js

```diff
@@ -79,14 +83,12 @@ function convertResults(spectralResults, { config, logger }) {
     finalResultsObject.hasResults = true;
     finalResultsObject[severity].summary.total++;
 
-    if (!summaryOnly) {
```

Comment:
We don't want to filter this here anymore, because we want to be able to use the rule violations to compute the quality scores. This filtering is moved to the end of `run-validator.js`, right before printing output.